README.adoc

```diff
@@ -6,6 +6,133 @@ Rocket implementation of the Application Container Specification.
 The goal is to make Rocket available as an installable package for any
 distribution which uses RPM as the binary distribution mechanism.
 
+A further goal is to get the packaging files incorporated into the
+upstream source tree for the Rocket process so that packaging is
+considered a first-class part of the development and build process.
+
+== Package Files
+
+The Rocket package will consist of three (3) binary files and a set of
+supporting files and directories.  The placement of the files will be
+consistent with the
+https://fedoraproject.org/wiki/Packaging:Guidelines[Fedora Packaging
+Guidelines].
+
+```
+/usr/bin/actool
```

Comment:
ohh and yup ... maybe it's better for a single rpm to provide both, but some purists would probably complain about the separate upstreams and require actool be provided via appc/spec only